SessionMessagingKit/Open Groups/OpenGroupAPI.swift

```diff
@@ -839,9 +847,10 @@ public enum OpenGroupAPI {
         fileId: String,
         roomToken: String,
         authMethod: AuthenticationMethod,
-        using dependencies: Dependencies
+        using dependencies: Dependencies,
+        skipAuthentication: Bool = false
```

Comment:
`using dependencies` should always be the last parameter - please put `skipAuthentication` before it